cmd/operator/server.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/config"
+	"github.com/sirupsen/logrus"
+)
+
+// makeRolloutHandler creates a request handler to perform a rollout process.
+func makeRolloutHandler(logger *logrus.Logger, cfg *config.Config) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		ctx := req.Context()
+		services, err := getTargetedServices(ctx, logger, cfg.Targets)
+		if err != nil {
+			logger.Errorf("failed to get targeted services: %v", err)
+			w.WriteHeader(http.StatusInternalServerError)
+			fmt.Fprintf(w, "failed to retrieved services %v", err)
+			return
+		}
+		if len(services) == 0 {
+			logger.Warn("no service matches the targets")
+		}
```

Comment:
move this into runRollouts because it's (1) duplicated (2) not handled differently than any error right now

Reply:
I agree with this, but I just realized that if we do this, `runDaemon` [can't ever return an error](https://github.com/GoogleCloudPlatform/cloud-run-release-operator/blob/c6b0627606cca06a83a0828fecce8b5042f3cadf/cmd/operator/main.go#L156). 

But this is not a big problem since getting services mostly fail either because 
1. a Cloud Run region is down (like last week) 
2. wrong region is specified (e.g. `us-east`, notice typo)

For 1, we can make the argument that it's not the tool's problem and keeping it running is actually the right decision. For 2, the user should quickly realize of the problem since it'd happen for the first time right at the beginning of the execution.